src/data/customer/Order.js

```diff
@@ -55,7 +55,33 @@ export const getOrdersWithFiltersAsync = async (
 };
 
 /**
- * Retrieve All Order Counts
+ * Retrieve Remaining Orders Count
+ * @param {object} filterObject # {pickupDate, withOrder}
```

Comment:
@maartenhemmes  Do i need to pass statusIds: 2 in the filter object ? 

as count should be only for the validated records.

if thats the case then please update docs.

Reply:
the orders in grouping_location don't have a status? @vinay3206